examples/experimental/onnx/mobilenet_v2/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
Do we really need __init__.py here?

Reply:
Good question! It is not a part of the nncf package. I will remove it.